pkg/parsers/operatingsystem/operatingsystem_unix.go

```diff
@@ -1,17 +1,24 @@
+// +build freebsd darwin !linux !windows
+
```

Comment:
And stuff with `!` here and it worked for me. And I believe it's how it should be. If windows tag doesn't work properly - it's some bug in our build system or somewhere else.